### PR TITLE
Freestyle introduction

### PR DESCRIPTION
**Pull request by craigcwaveformlabs**
Add "Introduction to FreeStyle" content. Also pulling in a stray Critique page that was lost outwith the navigation.

### CODE REVIEW

**Comment by craigcwaveformlabs** [comment]
@summers1981FA do you have a moment to check over this FreeStyle docs update to the Introduction page. Simply pulling in the document written by Roan.

File: docs/index.html [hunk 1]
```diff
@@ -9,11 +9,9 @@
       <div class="u-padding-bottom--large u-padding-top--large">
         <div class="Hero-body">
           <p class="Hero-leader">
-           Design at FreeAgent
-          </p>
-          <p>
-            Current build: <a href="{{ site.github.repo }}/releases">{% project_version %}</a>
+           FreeStyle
           </p>
+          <p>FreeStyle is a design system that aims to unite the look, feel, tone and experience of our products, brand and marketing material. It’s a resource for anyone involved in creating content, marketing collateral, product features or other user experiences.</p>
```
Comment (summers1981FA) [inline docs/index.html]: @craig when reviewing the design I noticed that it is just a standard font size. I wondered whether this message should have a more weight and be a bigger font size...ex http://freestyle.freeagent.io/utilities/font-size/#xx-large

Comment (summers1981FA) [reply docs/index.html]: It also looks like the line-height might not be matching that of the docs.

**Comment by craig** [comment]
# BIG LIKE THIS IS OKAY??? COME ON!

**Comment by craigcwaveformlabs** [comment]
@summers1981FA i've updated the index page to be bolder on the introductory paragraph. i think it still needs work but ii'll probably work that into my FreeStyle-IA branch as a bigger change.

**Comment by summers1981FA** [comment]
Looks good to me so thumbs up. 

As a separate update I personally would bump up the FreeStyle Masthead since the intro paragraph has a similar presence.

**Pull request merged**